frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -44,6 +44,13 @@ interface SubscribedProps {
     keystores?: Array<{ type: 'hardware' | 'software' }>;
 }
 
+export type SharedPanelProps = {
+  // eslint-disable-next-line react/no-unused-prop-types
+  activeSidebar: boolean;
+  // eslint-disable-next-line react/no-unused-prop-types
+  sidebarStatus: string;
+}
```

Comment:
I wanted to ask if you could change sidebarStatus type from string to `'' | 'forceHidden'`...

but looking at it more closely: it seems to be only used to swip sidebar in/out on mobile, but it is set only by bitbox01.tsx 🤔  and BitBox01 is not supported on mobile 🤔 

It looks like it was initially added also for bitbox02, but BB02 doesn't need it anymore since setup changed to View component.
https://github.com/digitalbitbox/bitbox-wallet-app/pull/529

maybe lets keep as is and lets try to remove sidebarStatus in the future.

Reply:
True. The value of sidebarStatus is a bit unclear.